src/nav/ChatNavBar.js

```diff
@@ -31,36 +32,46 @@ export default function ChatNavBar(props: Props) {
       ? 'default'
       : foregroundColorFromBackground(backgroundColor);
 
+  const insets = useSafeAreaInsets();
+
   return (
-    <View
```

Comment:
(The default
    algorithm used to display the diff in `ChatNavBar` gives a lot of
    noise; `git diff -w` should reduce that quite a lot.)

It will, but perhaps this is a good time to plug a handy Git feature! This is one I relatively recently discovered and enabled in [my gitconfig](https://gist.github.com/gnprice/5f71b63464a76bd7e0480f207b1bbff3):
```
# Use color to identify code that was moved around verbatim.
[diff]
	# Enable the feature.
	colorMoved = zebra

	# Supercharge it to do the equivalent of `git diff -b`.
	# …
	colorMovedWS = ignore-space-change
```
(And `git diff -b` is a close relative of `-w`; they're pretty similar but I tend to find `-b` is closer to what I'm really looking for in principle.)

So here's how the main part of that diff comes out with my normal `git usp`, without going back and adding any other flags:

![image](https://user-images.githubusercontent.com/28173/106333286-2a788280-623d-11eb-8767-fb291b4ba3e9.png)

The "moved" colors let me see immediately that the code in each of those chunks didn't change beyond getting indented. Then I can match up the chunks by eye, and pretty quickly isolate what's changed. Not as simple as the `git diff -b` or `-w` view, but simple enough that I usually don't feel the need anymore to go reach for that one.

(The default colors are different from those and I think less helpful; those colors are also in my posted gitconfig.)

Reply:
Hmm, interesting! 

> Not as simple as the `git diff -b` or `-w` view, but simple enough that I usually don't feel the need anymore to go reach for that one.

It is quite a lot more lines than `git diff -b`. I'm not sure how long it'll take until I see this output as simpler, but I'll give it a go and find out! 🙂

Reply:
To be clear, the output of `git diff -b` is definitely simpler than this!

I just mean that this is simpler than reading the plain diff (and having to look closely to spot the few bits that changed); and it's enough simpler, giving me enough of what I'd get from `git diff -b`, that when I'm in the middle of a review I typically don't feel the need to go reach for `git diff -b`.

(And I don't just have `-b` or `-w` on all the time, because I do want to know about the changes those ignore; when they're relevant, I want to both see those changes, and see the changes with those simplified away, just separately.)